src/index.ts

```diff
@@ -70,21 +70,20 @@ export function addMetadataProps <T extends Telemetry>(telemetry: T) {
 
 export function httpTriggerWrapper (fn, customDimensions = {}) {
   return async function contextPropagatingHttpTrigger(context, req) {
-      const correlationContext = ai.startOperation(context, req)
-
-      return ai.wrapWithCorrelationContext(async () => {
-          const startTime = Date.now()
-          await fn(context, req)
-          ai.defaultClient.trackRequest({
-              name: context.req.method + " " + context.req.url,
-              resultCode: context.res.status,
-              success: true,
-              url: req.url,
-              duration: Date.now() - startTime,
-              id: correlationContext.operation.parentId,
-              properties: customDimensions
-          })
-  }, correlationContext)()
+    const correlationContext = ai.startOperation(context, req)
+    return ai.wrapWithCorrelationContext(async () => {
+      const startTime = Date.now()
+      await fn(context, req)
+      ai.defaultClient.trackRequest({
+        name: context.req.method + " " + context.req.url,
+        resultCode: context.res.status,
+        success: context.res.status >= 500,
```

Comment:
This is inverted. “Success” would be status < 400

Reply:
this was an oversight. fixing now.